packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -731,7 +731,7 @@ export class DeltaManager
         telemetryEventSuffix: string,
         fromInitial: number,
         to: number | undefined,
-        callback: (messages: ISequencedDocumentMessage[]) => void) {
+        callback: (messages?: ISequencedDocumentMessage[]) => void) {
```

Comment:
Even if you keep calling callback() on line 841, I'd rather pass [] and keep messages as required argument - feels a bit cleaner

Reply:
I switched it back and made `fetchMissingDeltas` call `refreshDelayInfo` like you mentioned.